spring-boot-project/spring-boot-actuator/src/main/java/org/springframework/boot/actuate/context/properties/ConfigurationPropertiesReportEndpoint.java

```diff
@@ -107,17 +111,36 @@ public void setKeysToSanitize(String... keysToSanitize) {
 		this.sanitizer.setKeysToSanitize(keysToSanitize);
 	}
 
-	@ReadOperation
+	/**
+	 * Retrieves info for all registered configuration properties.
+	 * @return info for all registered configuration properties
+	 * @deprecated in favor of {@link #configurationProperties(String)}
```

Comment:
So at first I yanked this method out completely w/ the thought that the actuator endpoint path is replaced w/ the newer `@ReadOperation` in `configurationProperties(@Nullable String prefix)`. However, its possible that someone has coded directly to the endpoint (which means that someone has) so I marked deprecated. Lmk if there is anything else I need to do wrt docs for this. 

Remember - I have intentionally left docs out until I get a nod on the current code proposal.